__tests__/Auth0Client.test.ts

```diff
@@ -452,6 +452,29 @@ describe('Auth0Client', () => {
       );
     });
 
+    it('calls `tokenVerifier.verify` with the organization id', async () => {
+      const auth0 = setup({ organization: 'test_org_123' });
+
+      await loginWithPopup(auth0);
+
+      expect(tokenVerifier).toHaveBeenCalledWith(
+        expect.objectContaining({
+          organizationId: 'test_org_123'
+        })
+      );
+    });
+
+    it('calls `tokenVerifier.verify` with the organization id given in the login method', async () => {
+      const auth0 = setup();
+      await loginWithPopup(auth0, { organization: 'test_org_123' });
+
+      expect(tokenVerifier).toHaveBeenCalledWith(
+        expect.objectContaining({
+          organizationId: 'test_org_123'
+        })
+      );
+    });
```

Comment:
Should we test loginWithRedirect here aswell ?

Reply:
I thought I had covered everything but let me check 👍

Reply:
It is covered in index.test.ts. But I was confused why these tests ended up here and the others are in index.test.ts...

Reply:
Ah yes, to follow up on this, `loginWithRedirect` just defers all its logic to `buildAuthorizeUrl`, which has tests for this.